src/counts.jl

```diff
@@ -199,6 +231,24 @@ end
 
 countmap{T}(x::AbstractArray{T}) = addcounts!(Dict{T,Int}(), x)
 countmap{T,W}(x::AbstractArray{T}, wv::WeightVec{W}) = addcounts!(Dict{T,W}(), x, wv)
+"""
+    countmap(x::AbstractArray{T}, wv::WeightVec{W})
+
+### Args:
+* `x`: An `AbstractArray`.
```

Comment:
Repeating the type info from the signature isn't useful. You should say what's the meaning of this argument instead.